alacritty/src/logging.rs

```diff
@@ -129,6 +129,26 @@ impl log::Log for Logger {
     fn flush(&self) {}
 }
 
+fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
+    let now = time::strftime("%F %T.%f", &time::now()).unwrap();
+    let mut msg = format!("[{}] [{:<5}] [{}] ", now, record.level(), target);
+
+    // Alignment for the lines after the first new line character in the payload. We don't deal
+    // with fullwidth/unicode chars here, so just `msg.len()` will work fine.
+    let alignment = msg.len();
+
+    // Push lines with adding extra padding on the next line, which is trimmed later.
+    let lines = record.args().to_string();
+    for line in lines.split('\n') {
+        let line = format!("{}\n{:width$}", line, "", width = alignment);
+        msg.push_str(&line);
+    }
+
+    // Drop extra trailing alignment.
+    msg.truncate(msg.len() - alignment);
```

Comment:
Wouldn't this still include `\n`? Since alignment is just the whitespace?

Reply:
it's a lot of whitespaces though.

Reply:
Unless I'm missing an off by one somewhere, it seems to me like it should though: 
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=149234e85155df110bd356d7a7fa2e44

Reply:
Oh, just double-checked the old code.

Messages are expected to end in a newline, so it works out. Or at the very least it doesn't seem to matter.

Reply:
They must end with a new line, otherwise they won't be printed at all or something like that. The only goal is to remove trailing whitespaces here, though, you could use `rposition` to find `\n`, but doesn't really matter...